src/db/commit.rs

```diff
@@ -105,6 +105,18 @@ impl Commit {
         self.commit().value_hash().unwrap()
     }
 
+    pub fn next_mutation_id(&self) -> u64 {
+        let meta = self.commit().meta().unwrap();
+        let mid = match meta.typed_type() {
+            commit::MetaTyped::LocalMeta => meta.typed_as_local_meta().unwrap().mutation_id(),
```

Comment:
Do you have creating a genesis snapshot when we create the db on your list of todos?

Reply:
Nope! https://github.com/rocicorp/repc/issues/102